frameworks/TPOT/exec.py

```diff
@@ -4,12 +4,12 @@
 import sys
 import tempfile as tmp
 
+os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
 if sys.platform == 'darwin':
     os.environ['OBJC_DISABLE_INITIALIZE_FORK_SAFETY'] = 'YES'
-os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
-os.environ['OMP_NUM_THREADS'] = '1'
-os.environ['OPENBLAS_NUM_THREADS'] = '1'
-os.environ['MKL_NUM_THREADS'] = '1'
+    os.environ['OMP_NUM_THREADS'] = '1'
+    os.environ['OPENBLAS_NUM_THREADS'] = '1'
+    os.environ['MKL_NUM_THREADS'] = '1'
```

Comment:
@weixuanfu, I need approval from author, thanks

Reply:
Weixuanfu can correct me if I'm wrong, but with [TPOT parallelization](https://github.com/EpistasisLab/tpot/blob/6448bdb71ba08b4a0447c640d2f05a05e1affc21/tpot/base.py#L1545-L1574) the reason for setting this is the same as with auto-sklearn (this parallelization does not control further parallelization of underlying packages which may lead to using a number of cores exceeding the specified limit with `n_jobs`).